src/xopen/__init__.py

```diff
@@ -196,12 +196,19 @@ def __init__(self, path, mode='r', threads=None):
         """
         if mode not in ('r', 'rt', 'rb'):
             raise ValueError("Mode is '{0}', but it must be 'r', 'rt' or 'rb'".format(mode))
-        if threads is None:
-            threads = min(_available_cpu_count(), 4)
-        
+
         pigz_args = ['pigz', '-cd', path]
-        if threads != 0:
+        if threads:  # Check if threads is not 0 or None.
```

Comment:
This is a redundant comment. It should be removed.

Reply:
I extremely dislike the implicitness of truthy/falsey testing. In this case the possible value is an optional integer that may be `0` or `None`. I do not want to test for `[]`, `{}`, `()`, `False` or empty set. Hence the comment to clarify what values I am expecting. 
But this is a style thing and just my personal taste, so I will remove it.

Reply:
Thanks.